src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -87,11 +86,13 @@ public void run(RunConfiguration config) throws AppEngineException {
               .getAbsolutePath();
       jvmArguments.add("-javaagent:" + appengineAgentJar);
     } 
-    arguments.addAll(DevAppServerArgs.get("server", config.getHost()));
-    arguments.addAll(DevAppServerArgs.get("address", config.getPort()));
-    arguments.addAll(DevAppServerArgs.get("jvm_flag", config.getJvmFlags()));
+    arguments.addAll(DevAppServerArgs.get("address", config.getHost()));
+    arguments.addAll(DevAppServerArgs.get("port", config.getPort()));
+    if (config.getJvmFlags() != null) {
+      jvmArguments.addAll(config.getJvmFlags());
+    }
 
-    arguments.add("--allow_remote_shutdown");
+    // arguments.add("--allow_remote_shutdown");
```

Comment:
right, jvm_flag is a dev_appserver2 thing. But we have access directly to the jvm here, so that's okay.

Does stopping the server remotely still work even without allow_remote_shutdown?